### ISSUE TITLE
Web app web view crash or (null) when click reload on not fully loaded page

### ISSUE THREAD

**Issue by Viiprogrammer** [telegramdesktop/tdesktop#27911]
### Steps to reproduce

1. Open webapp
2. Click before fully loaded
3. Get (null) or crash

https://github.com/telegramdesktop/tdesktop/assets/17622604/1e07a3a5-7005-440b-a08a-7a02e2abc60f

### Expected behaviour

Works OK

### Actual behaviour

Crash / null

### Operating system

Ubuntu 22.04.4 LTS x86_64

### Version of Telegram Desktop

5.0.1

### Installation source

Static binary from official website

### Logs

```shell
QPainter::begin: Paint device returned engine == 0, type: 2
QWidget::render: Cannot render with an inactive painter
[libvpx-vp9 @ 0x723c04807000] v1.13.0
[libvpx-vp9 @ 0x723c04807000] v1.13.0
[libvpx-vp9 @ 0x723c04807400] v1.13.0
[libvpx-vp9 @ 0x723c04807400] v1.13.0
qt.waylandcompositor.hardwareintegration: Failed to initialize EGL display. Could not get EglDisplay for window.

(Telegram:168717): GLib-GIO-WARNING **: 05:24:26.511: Your application does not implement g_application_activate() and has no handlers connected to the 'activate' signal.  It should do one of these.
QWindow::setWindowStates does not accept Qt::WindowActive

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.715: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.715: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.716: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.717: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.722: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.723: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.723: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.723: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168717): Gdk-CRITICAL **: 05:24:26.725: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed
qt.waylandcompositor.hardwareintegration: Failed to initialize EGL display. Could not get EglDisplay for window.

(Telegram:168784): GLib-GIO-WARNING **: 05:24:32.800: Your application does not implement g_application_activate() and has no handlers connected to the 'activate' signal.  It should do one of these.

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.897: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.898: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.898: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.898: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.903: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.904: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.904: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.904: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168784): Gdk-CRITICAL **: 05:24:32.907: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed
qt.waylandcompositor.hardwareintegration: Failed to initialize EGL display. Could not get EglDisplay for window.

(Telegram:168843): GLib-GIO-WARNING **: 05:24:35.509: Your application does not implement g_application_activate() and has no handlers connected to the 'activate' signal.  It should do one of these.

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.627: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.627: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.628: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.628: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.632: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.633: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.633: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.633: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed

(Telegram:168843): Gdk-CRITICAL **: 05:24:35.635: gdk_monitor_get_scale_factor: assertion 'GDK_IS_MONITOR (monitor)' failed
```

**Comment by ilya-fedin** [comment]
Hm, I sure this was fixed in the past

**Comment by Viiprogrammer** [comment]
> Hm, I'm sure this was fixed in the past

Most of the crashes have been fixed, but I didn't have time to report on the last one again.

**Comment by ilya-fedin** [comment]
I mean, exactly the same report was fixed: #27481

**Comment by Viiprogrammer** [comment]
> I mean, exactly the same report was fixed: #27481

Hmm, maybe not fully 🧐 But it is really present in 5.0.1 and some previous
![image](https://github.com/telegramdesktop/tdesktop/assets/17622604/404286aa-3e38-448a-b009-5799e3ded6a8)
